src/com/company/Homework3/ShipBattle.java

```diff
@@ -0,0 +1,156 @@
+package com.company.Homework3;
+
+import java.util.Scanner;
+
+public class ShipBattle {
+    public static void main(String[] args) {
+        System.out.println("This is beautiful game Ship Battle \n Let's start ");
+        //createEmptyBoardAndShow();
+        System.out.println("Please include coordinates !!");
+        System.out.println("Format including: type number 0-10 and press enter, do this action 2 time for X and Y coordinates ");
+        int[][] battleBoard = createShipsInBoard(createShipsPoints());
+        String[][] resultBoard = checkCoordinates(includeCoordinates(),battleBoard, createEmptyBoardAndShow());
+
+
+        for (int i = 0; i < 100; i++) {
+            if (finishCheck(resultBoard)) {
+                checkCoordinates(includeCoordinates(), battleBoard, resultBoard);
+            } else {
+                System.out.println("FINISH!!!!");
+                break;
+            }
+        }
+
+    }
+
+    public static String[][] createEmptyBoardAndShow() {
+        String[][] emptyArray = new String[11][11];
+        for (int i = 0; i < 11; i++) {
+            for (int j = 0; j < 11; j++) {
+                if (i == 0) {
+                    emptyArray[i][j] = "" + j;
+                } else if (j == 0) {
+                    emptyArray[i][j] = "" + i;
+                } else {
+                    emptyArray[i][j] = "*";
+                }
+                System.out.print(emptyArray[i][j] + " ");
+            }
+            System.out.println();
+        }
+        return emptyArray;
+
+    }
+
+
+    public static int[] createShipsPoints() {
+        int ship6 = (int) (Math.random() * 11);
+        int ship5_1 = (int) (Math.random() * 11);
+        int ship5_2 = (int) (Math.random() * 11);
+        int ship3_1 = (int) (Math.random() * 11);
+        int ship3_2 = (int) (Math.random() * 11);
+
+
+        int[] ships = new int[7];
+
+        while ((ship6 == 0)) {
+            ship6 = (int) (Math.random() * 11);
+        }
+        while ((ship5_2 == ship6) || (ship5_2 == 0) || (ship5_2 <= ship6 + 1 && ship5_2 >= ship6 - 1)) {
+            ship5_2 = (int) (Math.random() * 11);
+        }
+        while (ship5_1 == ship6 || ship5_1 == ship5_2 || ship5_1 < 8) {
+            ship5_1 = (int) (Math.random() * 11);
+        }
+        while (ship3_2 == ship6 || ship3_2 == ship5_2 || ship3_2 == ship5_1 || ship3_2 == 0 || ((ship3_2 <= ship6 + 1 && ship3_2 >= ship6 - 1) || (ship3_2 <= ship5_2 + 1 && ship3_2 >= ship5_2 - 1))) {
+            ship3_2 = (int) (Math.random() * 11);
+        }
+        while (ship3_1 == ship6 || ship3_1 == ship5_2 || ship3_1 == ship3_2 || ship3_1 == 0
+                || ((ship3_1 <= ship6 + 1 && ship3_1 >= ship6 - 1) || (ship3_1 <= ship5_2 + 1 && ship3_1 >= ship5_2 - 1) || (ship3_1 <= ship3_2 + 1 && ship3_1 >= ship3_2 - 1))) {
+            ship3_1 = (int) (Math.random() * 11);
+        }
```

Comment:
Too Complex Code is not readable

Reply:
Maybe you want me to add comments to the functions?